app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = I18n.t "base_title"
+    if page_title.empty?
```

Comment:
So sánh empty?, blank? , nil?

Reply:
empty? -> true nếu không có phần tử nào(chuỗi rỗng, mảng rỗng, hash rỗng).
blank? -> true nếu chuỗi rỗng hoặc chứa toàn khoảng trắng hoặc object false, nil.
nil? -> true chỉ với đối tượng nil.